extension/src/experiments/webview/messages.ts

```diff
@@ -226,12 +226,13 @@ export class WebviewMessages {
     }
   }
 
-  private async changeHasMoreOrLessCommits() {
+  private async changeHasMoreOrLessCommits(update?: boolean) {
     const availableNbCommits = await this.getNumCommits()
     const nbOfCommitsToShow = this.experiments.getNbOfCommitsToShow()
     this.hasMoreCommits = availableNbCommits > nbOfCommitsToShow
     this.isShowingMoreCommits =
       Math.min(nbOfCommitsToShow, availableNbCommits) > 1
+    update && this.sendWebviewMessage()
```

Comment:
Bug appears to be fixed (If I'm understanding the issue correctly), but out of curiosity, how does this change fix the bug? I see this function called twice but neither call adds any arguments to the function call 🤔

Reply:
Thanks, it is due to a bad merge. Will fix ASAP